com.wamas.ide.launching/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,7 @@ Require-Bundle: org.eclipse.xtext,
  org.eclipse.xtend.lib,
  org.eclipse.emf.common,
  org.eclipse.core.resources,
- org.eclipse.pde.core;bundle-version=3.15.100,
+ org.eclipse.pde.core;bundle-version="3.18.100",
```

Comment:
3.17.200.v20231118 should be the first one containing the underlying code change f2b9ed797620fc5cad2a25ca27d289fee5961ac9, therefore 3.17.200 might be better for backwards compatibility